R/liftover.R

```diff
@@ -0,0 +1,287 @@
+#' @title Liftover Bedpe.
+#'
+#' @description Use liftOver to convert a bedpe file between the two main genome builds (grch37/hg38).
```

Comment:
Just `liftover`. In addition, this function is no longer restricted to only bedpe files.

Reply:
Updated!